src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Wrapper,
+  Divider,
+  Sections,
+  NavSections,
+  SectionTitle,
+  SocialMedia,
+  SocialMediasContainer,
+  LogoImage,
+  SocialMediaName,
+} from "./Footer.styles";
+import { Icon } from "../Icon/Icon";
+import { channels, sections } from "@/utils/constants";
+
+const Footer = () => {
+  return (
+    <Wrapper>
+      <LogoImage id="logo-sudene" width={311} height={100} />
+      <Sections>
+        <NavSections>
+          {Object.entries(sections).map(([key, item]) => (
+            <SectionTitle key={key} href={item.path || ""}>
+              {item.name}
+            </SectionTitle>
+          ))}
+        </NavSections>
+        <Divider />
+        <SocialMediasContainer>
+          {channels.map(({ href, icon, size, name }, index) => (
+            <SocialMedia target="_blank" key={index} title={href} href={href}>
+              <Icon id={icon} size={size} />
+              <SocialMediaName>{name}</SocialMediaName>
+            </SocialMedia>
```

Comment:
The social media component isn't aligning the icon with text, here:

![image](https://github.com/user-attachments/assets/6f7df7b1-c027-4184-88ba-2e687468808a)